eclair-core/src/main/scala/fr/acinq/eclair/channel/fsm/Channel.scala

```diff
@@ -251,8 +252,13 @@ class Channel(val nodeParams: NodeParams, val wallet: OnChainChannelFunder with
       context.system.eventStream.publish(ChannelRestored(self, data.channelId, peer, remoteNodeId, data))
       txPublisher ! SetChannelId(remoteNodeId, data.channelId)
 
-      // we watch all unconfirmed funding txs, whatever our state is
-      // (there can be multiple funding txs due to rbf, and they can be unconfirmed in any state due to zero-conf)
+      // We watch all unconfirmed funding txs, whatever our state is.
+      // There can be multiple funding txs due to rbf, and they can be unconfirmed in any state due to zero-conf.
+      // To avoid a herd effect on restart, we had a delay before watching funding txs.
```

Comment:
```suggestion
      // To avoid a herd effect on restart, we add a delay before watching funding txs.
```

Reply:
Doh, fixed in https://github.com/ACINQ/eclair/pull/2733/commits/0966d8064da67b9642d02728a011a0b6d306bcdd